ui/lib/kv/addon/components/page/secret/paths.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { kvMetadataPath, kvDataPath } from 'vault/utils/kv-path';
+
+/**
+ * @module KvSecretPaths is used to display copyable secret paths to use on the CLI and API.
+ *
+ * <Page::Secret::Paths
+ *  @path={{this.model.path}}
+ *  @secret={{this.model.secret}}
+ *  @breadcrumbs={{this.breadcrumbs}}
+ * />
+ *
+ * @param {model} secret - Ember data model: 'kv/data', the new record for the new secret version saved by the form
+ * @param {string} path - kv secret path
+ * @param {array} breadcrumbs - Array to generate breadcrumbs, passed to the page header component
+ */
+
+export default class KvSecretPaths extends Component {
+  @service namespace;
+
+  get paths() {
+    const { backend, path } = this.args.secret;
+    const namespace = this.namespace.path;
```

Comment:
I wonder if this will need to be encoded for the API paths 🤔

Reply:
Oh good question - I just checked and namespace can only include `-`, `.` and `_` so I think we're good!
<img width="604" alt="Screenshot 2023-08-22 at 2 56 18 PM" src="https://github.com/hashicorp/vault/assets/68122737/d9ece377-4d0d-4f48-874b-dc2e75cb43d9">